WcaOnRails/app/helpers/persons_helper.rb

```diff
@@ -33,8 +33,15 @@ def return_podium_class(result)
   end
 
   def delegate_badge(kind)
-    content_tag(:span, class: "badge delegate-badge") do
-      link_to(t("enums.user.delegate_status." + kind), "/delegates", title: t("enums.user.delegate_status." + kind), data: { toggle: "tooltip", placement: "bottom" })
+    title = t("enums.user.delegate_status." + kind)
+    class_list = "badge delegate-badge"
+    data = { toggle: "tooltip", placement: "bottom" }
+    if kind.eql? "trainee_delegate"
+      content_tag(:span, title, class: class_list, data: data)
+    else
+      content_tag(:span, class: class_list) do
```

Comment:
I believe there is some better way to handle this, but I was not able to figure it out. Please let me know if there is a better way. 😬

Reply:
I'm pretty sure you can keep the tooltip on the span, so you can probably do something like that:
```
title = t("enums.user.delegate_status." + kind)
content_tag(:span, class: "badge delegate-badge", data: { toggle: "tooltip", placement: "bottom" }, title: title) do
  kind == "trainee_delegate" ? title : link_to(title, "/delegates")
end
```